CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ UNRELEASED
 ----------
 
 * [ [#951](https://github.com/digitalfabrik/integreat-cms/issues/951) ] Add possibility to create categories for POIs
+* [ [#1745](https://github.com/digitalfabrik/integreat-cms/pull/1745) ] Added last modified date to media sidebar
```

Comment:
```suggestion
* [ [#1742](https://github.com/digitalfabrik/integreat-cms/issues/1742) ] Add last modified date to media sidebar
```
Just a few small details:
- Typically, we link issues instead of PRs in the changelog
- If no issue exists, we link the PR id but treat it like an issue id, because the [bash command](https://github.com/digitalfabrik/integreat-cms/blob/a10d0b86986d3bce9cf0a1e981a44a4e38333eea/.circleci/config.yml#L366) we use for processing the changelog is very fragile (probably it has a male ego) and cannot handle `pull` links - instead, we rely on Github to redirect e.g. from `https://github.com/digitalfabrik/integreat-cms/issues/1745` to `https://github.com/digitalfabrik/integreat-cms/pull/1745`
- We try to use the [imperative mood](https://cbea.ms/git-commit/#imperative) for commit messages and changelog entries